src/commands/build.ts

```diff
@@ -27,6 +28,11 @@ export default defineCommand({
   },
   async run(ctx) {
     overrideEnv('production')
+    if (!isCompatibleNodeVersion()) {
+      consola.warn(
```

Comment:
```suggestion
      console.warn(
```

Reply:
@dreyfus92  consola is right, its the library of unjs.